### DISCUSSION TITLE
Flexget rss torrent download with delay

### DISCUSSION THREAD

**Discussion by geotibi** [Flexget/Flexget#3541] [Q&A]
Hello,

I've installed Flexget and configured a task to run every 30 seconds. The issue that I'm facing is that if the RSS feed is updated at 12:00:00 and the task is running at 12:00:00, 12:00:30, 12:01:00, 12:01:30 ..... the new torrents are automatically downloaded after around 4 minutes, so when the task is run at 12:04:00 even if I can see that the rss feed is updated when accessing manually the rss feed by a web browser. So, even if I scheduled the task every 30 seconds and I can see in the tasks that is running every 30 seconds, the "real" schedule is at around 4 minutes.

Did someone faced this kind of issue that the file are downloaded with delay, after task is executed few times?

**Comment by crawln45** [comment]
There is nothing wrong with how flexget it's processing the feed. Every 30 seconds is excessive at the least as most rss feeds don't update but once every 15 minutes. Rss feeds are cached and only the stag is reviewed to see if an update is required during the running process if I remember correctly. Running against an rss feed should not be attempted in less than 5 minute increments. This is out of politeness for the rss feed and flexget usually won't process anything before the 2 min mark.

**Comment by geotibi** [comment]
It's strange because I'm running same RSS feed at every minute from a crontab for Synology download station and is working.
